src/features/ITGovernance/Admin/Actions/RequestHome/index.tsx

```diff
@@ -0,0 +1,252 @@
+import React, { useState } from 'react';
+import { useTranslation } from 'react-i18next';
+import { Link, useHistory } from 'react-router-dom';
+import { Grid, Icon } from '@trussworks/react-uswds';
+import {
+  GetSystemIntakeContactsDocument,
+  SystemIntakeFragmentFragment,
+  useDeleteSystemIntakeContactMutation,
+  useGetSystemIntakeContactsQuery
+} from 'gql/generated/graphql';
+
+import IconButton from 'components/IconButton';
+import PageHeading from 'components/PageHeading';
+import ReviewRow from 'components/ReviewRow';
+import SystemIntakeContactsTable from 'components/SystemIntakeContactsTable';
+import TaskStatusTag from 'components/TaskStatusTag';
+import useMessage from 'hooks/useMessage';
+
+import { DefinitionCombo } from '../../Decision';
+
+const RequestHome = ({
+  systemIntake
+}: {
+  systemIntake: SystemIntakeFragmentFragment;
+}) => {
+  const { t } = useTranslation('requestHome');
+  const { showMessage, showErrorMessageInModal } = useMessage();
+
+  const [expandedOverview, setExpandedOverview] = useState(false);
+
+  const { data, loading } = useGetSystemIntakeContactsQuery({
+    variables: {
+      id: systemIntake.id
+    }
+  });
+
+  const [removeContact] = useDeleteSystemIntakeContactMutation({
+    refetchQueries: [
+      {
+        query: GetSystemIntakeContactsDocument,
+        variables: { id: systemIntake.id }
+      }
+    ]
+  });
+
+  const handleRemoveContact = (id: string) =>
+    removeContact({ variables: { input: { id } } })
+      .then(() => {
+        showMessage(t('requestHome:pocRemoval.success'), {
+          type: 'success'
+        });
+      })
+      .catch(() => {
+        showErrorMessageInModal(t('requestHome:pocRemoval.error'));
+      });
+
+  const history = useHistory();
+
+  const bizCaseStatusToRender = ['DONE', 'NOT_NEEDED', 'SUBMITTED'].includes(
+    systemIntake.itGovTaskStatuses.bizCaseDraftStatus
+  )
+    ? systemIntake.itGovTaskStatuses.bizCaseFinalStatus
+    : systemIntake.itGovTaskStatuses.bizCaseDraftStatus;
+
+  return (
+    <div data-testid="request-home">
+      <div className="margin-bottom-6">
+        <PageHeading className="margin-y-0">
+          {t('requestHome.title')}
+        </PageHeading>
+        <p className="easi-body-medium margin-y-0">
+          {t('requestHome.description')}
+        </p>
+      </div>
+
+      {/* Project team and POC section */}
+      <div className="margin-y-4 padding-bottom-6 border-bottom-1px border-base-light">
+        <h2 className="margin-bottom-0">
+          {t('requestHome.sections.teamInfo.heading')}
+        </h2>
+        <p className="easi-body-medium margin-top-0 margin-bottom-2">
```

Comment:
Same as the text above - needs larger top margin.

```suggestion
        <p className="easi-body-medium margin-top-05 margin-bottom-2">
```